addons/survey/models/survey_question.py

```diff
@@ -261,7 +270,19 @@ def _compute_validation_required(self):
             if not question.validation_required or question.question_type not in ['char_box', 'numerical_box', 'date', 'datetime']:
                 question.validation_required = False
 
-    @api.depends('is_conditional')
+    @api.depends('survey_id', 'survey_id.question_ids')
+    def _compute_allowed_triggering_question_ids(self):
+        """ This field/function is required to fetch the right questions when
+        the child question does not yet exist as a question. """
+        for question in self:
+            question.allowed_triggering_question_ids = self.search([
+                ('survey_id', '=', (question.survey_id._origin.id
+                                    if isinstance(question.survey_id.id, models.NewId)
+                                    else question.survey_id.id)),
+                ('question_type', 'in', ['simple_choice', 'multiple_choice'])
+            ])
+
+    @api.depends('allowed_triggering_question_ids')
```

Comment:
Don't think this should depends on that. A computed stored that depends on a compute non stored.. And "YOU MADE 3 BUGS" is arriving au galop. :smile: (if you have the reference #website_event)
This should still depends on is_conditional IMO